roles/node-images-compute/tasks/metal.yml

```diff
@@ -39,3 +39,12 @@
     masked: "{{ item.masked | default(false) }}"
     state: "{{ item.state }}"
   loop: "{{ services }}"
+
+- name: Clear /etc/fstab
```

Comment:
This does remove the `/tmp` mount, we want to keep that.
Eventually all of our images will have it, NCNs don't have it at the moment because folks took advantage of it _not_ being `tmpfs` and now it's relied on too much.

CNs and UANs should keep using it.

Reply:
Fixed.